packages/analytics/src/DataStore.ts

```diff
@@ -0,0 +1,43 @@
+import merge from 'lodash/merge';
+/**
+ * This is a simple helper class to hold data in memory and encapsulates get/set data logic.
+ */
+class DataStore {
+  data: Record<string, unknown>;
```

Comment:
Could we store this into a type and reference it wherever it is used (get and set methods)?

Reply:
Yes, created a `Data` type for that. Thanks!